_posts/minutes/2016-11-23-23.md

```diff
@@ -0,0 +1,44 @@
+---
+    title: 'Minutes of Meeting on 23/11/2016'
+    author: qaisjp
+    category: minutes
+---
+This week's minutes were taken by Joshua Green.
```

Comment:
I suppose we could remove "This week's minutes were taken by.." and just use the author feature